assets/enemy.cfg.toml

```diff
@@ -10,7 +10,7 @@ range_melee_deaggro = 70.0
 range_ranged_melee = 29.0
 range_ranged_aggro = 100.0
 range_ranged_ranged = 100.0
-range_nearby = 20.0
+range_nearby = 50.0
```

Comment:
What if this change is the culprit of the freezes? It's something that stands out to me as I didn't dig deeper and I'm unsure about what it does.